### PR TITLE
added the config reloader

### PR DESCRIPTION
**Pull request by Kaali09**
This change includes a config reloader installation and it is required to pod restart when there are config map changes. 

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Deployed in dev and tested the script. 

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ansible/inventory/env/group_vars/all.yml [hunk 1]
```diff
@@ -204,3 +204,7 @@ content_port: 6379
 device_port: 6380
 user_port: 6381
 dialcode_port: 6382
+
+### docker image pull secrets name and this will be created in kubernetes
+imagepullsecrets: sunbird-registry-secret
```
Comment (keshavprasadms) [inline ansible/inventory/env/group_vars/all.yml:209]: Please update this to be env specific. Take a look at the sunbird-devops repo for example.
`imagepullsecrets: "{{ env }}registrysecret"`

**Review by keshavprasadms — CHANGES_REQUESTED** [review]
Please also rasie a PR to add the jenkins job config for this change in devops repo and link the PR here as a comment.

File: ansible/inventory/env/group_vars/all.yml (hunk 1 above)
Comment (Kaali09) [reply ansible/inventory/env/group_vars/all.yml]: updated as suggested.

**Comment by Kaali09** [comment]
This is the PR for jenkins job config changes. 
https://github.com/project-sunbird/sunbird-devops/pull/2470

**Comment by Kaali09** [comment]
> Please also rasie a PR to add the jenkins job config for this change in devops repo and link the PR here as a comment.

Have created the PR for jenkins job config changes. https://github.com/project-sunbird/sunbird-devops/pull/2470

**Review by keshavprasadms — APPROVED** [review]

**Review by anandp504 — APPROVED** [review]
Looks good :+1:

**Pull request merged**